docs/hydro/legacy/capacity_factor.rst

```diff
@@ -0,0 +1,10 @@
+Capacity Factor
++++++++++++++++
+EIA publishes US average monthly capacity factors for hydro plants. The `table
+<https://www.eia.gov/electricity/annual/html/epa_04_08_b.html>`_` is used to
+produce a profile for each hydro plant in the grid. Note that we are using the same set of capacity factor independently of the geographical location of the plant. As a result,
```

Comment:
Should this be 2 lines? Also should `capacity factor` be `capacity factors`?

Reply:
Added the (s) and split the paragraph in 2